keps/prod-readiness/sig-apps/4017.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 4017
+alpha:
+  approver: "@soltysh"
```

Comment:
this should be a PRR approver, @wojtek-t?

Reply:
updated